core/test/audits/byte-efficiency/byte-efficiency-audit-test.js

```diff
@@ -137,6 +137,17 @@ describe('Byte efficiency base audit', () => {
     assert.ok(failingResult.score < 0.5, 'scores failing wastedMs');
   });
 
+  it('should score negative wastedMs as perfect', () => {
+    const negativeResult = ByteEfficiencyAudit.createAuditProduct({
+      headings: baseHeadings,
+      items: [{url: 'http://example.com/', wastedBytes: -1 * 1000}],
+    }, graph, simulator, {gatherMode: 'timespan'});
+
+    assert.equal(negativeResult.score, 1);
+    assert.ok(negativeResult.numericValue < 0);
+    assert.equal(negativeResult.numericValue, negativeResult.details.overallSavingsMs);
```

Comment:
explicit test asserting that we want the negative result in the LHR even though it won't appear in the HTML report once it's tucked into the passing audits section (`displayValue` is used then)